scripts/build-syntaxes

```diff
@@ -1,9 +1,14 @@
-#!/bin/bash
-set -euo pipefail
+#!/usr/bin/env python
+import glob
+import os
+import sys
+import subprocess
 
-for SRC in syntaxes/ansible/*.plist; do
-
-    DST=$(basename "$(basename "$SRC" .plist)")
-    python3 syntaxes/plist2xml.py "$SRC" "syntaxes/ansible/generated/$DST"
-
-done
+for src in glob.glob("syntaxes/ansible/*.plist"):
+    dst = os.path.splitext(os.path.basename(src))[0]
+    cmd = [sys.executable, "syntaxes/plist2xml.py", src, f"syntaxes/ansible/generated/{dst}"]
```

Comment:
Why don't we just import this Python script? Using a subprocess seems too much.

Reply:
The plist2xml is a standalone util and the scope of the change was
to remove bash. I am not against calling the code directly but it should be a follow-up. Performance is not my concern considering that the entire conversion takes less a second and happens only at build time.

Reply:
Okay. I can live with that.